symbols/arm9.yml

```diff
@@ -1060,7 +1080,13 @@ arm9:
         
         r0: format
         ...: variadic
-    - name: GetDebugLogFlag
+    - name: InitDebugFlag2
+      address:
+        NA: 0x200C230
+      description: |-
+        Would have initialised the second set of debug flags.
+        Does nothing in release binary.
+    - name: GetDebugFlag2
```

Comment:
Hmm @End45 any thoughts on this name change? I'm just now remembering that `GetDebugFlag2` was the old name, before we renamed it in https://github.com/UsernameFodder/pmdsky-debug/pull/169. So this would actually be a reversion. @marius851000 do you think `DebugFlag2` is a more appropriate name than `DebugLogFlag`?

Reply:
I think this would be a regression. The flags were renamed because now we know what each set of flags is used for, I think it's better to keep it that way.
Maybe Marius was using the old names when decompiling the functions?

Reply:
I'll rename the flag back to `debug_log_flag` unless there are any objections.

Reply:
Yes. I used the old name. Sorry for that, my fault.